### PR TITLE
Adds new BOS gun and Pilot Officer loadouts (among other adjustments)

### PR DESCRIPTION
**Pull request by PlayerAdelheid**
### Senior Knight-Defender
The Knight-Captain can now choose to spawn with a **5mm SMG** instead of their useless police shotgun loadout.

### Pilot Officer loadouts
_**The Backup**_

- OA-93 5mm PDW
- Plasma grenade (x1)

_**Extended-Range Survival Kit**_

- 4.73mm SMG
- Smoke grenade (x2)

### Damage tweaks
Some semi-automatic weapons (Like the varmint rifle, ratslayer, rangemaster, and marksman carbine) now deal slightly higher baseline damage to accomodate for their lack of firerate.

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 1]
```diff
@@ -1380,6 +1382,43 @@
...
+
+/obj/item/gun/ballistic/automatic/assault_carbine/smg
+	name = "\improper 5mm machine pistol"
+	desc = "A German-made armor piercing PDW chambered for 5mm. It has the markings of the Brotherhood of Steel on its receiver."
+	icon_state = "mp7"
+	item_state = "uzi"
+	icon_prefix = "mp7"
+	icon = 'icons/fallout/objects/guns/ballistic.dmi'
+	slowdown = 0.12
+	autofire_shot_delay = 1.2
```
Comment (JasmineRickards) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1412]: This is WAY too fast for 5mm weapons.

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 2]
```diff
@@ -1380,6 +1382,43 @@
...
+/obj/item/gun/ballistic/automatic/assault_carbine/olympic
+	name = "OA-93 machine pistol"
+	desc = "A pistol-sized Colt assault carbine chambered for 5mm made by Olympic Arms, issued to Vertibird pilots. It's lighter, concealable and boasts a higher rate of fire - though the short barrel and lack of a buttstock make it a very poor choice for anything but boarding encounters."
+	icon_state = "olympic"
+	item_state = "assault_carbine"
+	icon_prefix = "olympic"
+	icon = 'icons/fallout/objects/guns/ballistic.dmi'
+	slowdown = 0.15
+	mag_type = /obj/item/ammo_box/magazine/m5mm
+	autofire_shot_delay = 0.9
```
Comment (JasmineRickards) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1394]: Way too fast in general

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 3]
```diff
@@ -921,6 +921,7 @@
 	suppressor_x_offset = 31
 	suppressor_y_offset = 15
 	fire_sound = 'sound/f13weapons/marksman_rifle.ogg'
+	extra_damage = 4
```
Comment (JasmineRickards) [inline code/modules/projectiles/guns/ballistic/automatic.dm:924]: marksman rifle buff? I can get behind it.

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 4]
```diff
@@ -964,6 +965,7 @@
 	scope_x_offset = 4
 	scope_y_offset = 11
 	fire_sound = 'sound/f13weapons/hunting_rifle.ogg'
+	extra_damage = 5
```
Comment (JasmineRickards) [inline code/modules/projectiles/guns/ballistic/automatic.dm:968]: Rangemaster buff is a little iffy.

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 5]
```diff
@@ -1454,6 +1493,26 @@
...
+	name = "\improper 4.7mm machine pistol"
+	desc = "An especially developed and modified machine pistol chambered for 4.73 caseless munitions, equipped with a West-Tek Marksman™ optic. It's loaded using helical magazines."
+	icon_state = "mp7m"
+	item_state = "uzi"
+	icon_prefix = "mp7m"
+	icon = 'icons/fallout/objects/guns/ballistic.dmi'
+	spread = 18
+	can_attachments = FALSE
+	recoil = 0.5 //No big frame to hold onto
+	autofire_shot_delay = 1.2
```
Comment (JasmineRickards) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1506]: Straight upgrade, bad idea.

File: code/modules/jobs/job_types/enclave.dm [hunk 6]
```diff
@@ -465,13 +465,33 @@
...
+		/datum/outfit/loadout/pilotemergency,			//4.7mm SMG
+		)
+
+/datum/outfit/loadout/pilotbackup
+	name = "The Backup"
+	suit_store = /obj/item/gun/ballistic/automatic/assault_carbine/olympic
+	backpack_contents = list(
+		/obj/item/ammo_box/magazine/m5mm = 1,
+		/obj/item/grenade/f13/plasma = 1,
+		)
```
Comment (JasmineRickards) [inline code/modules/jobs/job_types/enclave.dm:479]: I like these extra loadouts.

File: code/modules/jobs/job_types/bos.dm [hunk 7]
```diff
@@ -814,10 +814,10 @@ Senior Knight
 		)
 
 /datum/outfit/loadout/sknightb
-	name = "Knight-Defender"
+	name = "Senior Knight-Defender"
 	backpack_contents = list(
-		/obj/item/gun/ballistic/shotgun/police=1,
-		/obj/item/ammo_box/shotgun/buck=2,
+		/obj/item/gun/ballistic/automatic/assault_carbine/smg=1,
+		/obj/item/ammo_box/magazine/m5mm=2,
```
Comment (JasmineRickards) [inline code/modules/jobs/job_types/bos.dm:820]: The gun used here needs major adjusting.

File: code/modules/projectiles/guns/ballistic/automatic.dm (hunk 1 above)
Comment (PlayerAdelheid) [reply code/modules/projectiles/guns/ballistic/automatic.dm:1412]: I think it's fair given that it has a negative damage and AP bonus

File: code/modules/projectiles/guns/ballistic/automatic.dm (hunk 2 above)
Comment (PlayerAdelheid) [reply code/modules/projectiles/guns/ballistic/automatic.dm:1394]: With the 30 spread, it's effectively a bullet hose. So anything beyond 5 tiles is a no-go zone.

File: code/modules/projectiles/guns/ballistic/automatic.dm (hunk 5 above)
Comment (PlayerAdelheid) [reply code/modules/projectiles/guns/ballistic/automatic.dm:1506]: I could make reduced capacity mags.

**Pull request merged**